### DISCUSSION TITLE
Help understanding refresh_interval

### DISCUSSION THREAD

**Discussion by davama** [authelia/authelia#6326] [Q&A] [answered]
Hello,

We use an openldap backend to auth users and im trying to understand when this setting kicks in and what it does.

What we have:

```
refresh_interval: 5m
```

What I see:

After deleting the user from ldap, I still see some references to said user in the db:

doing a simple `sqlite .dump` i can see the user in several tables.

```
INSERT INTO identity_verification VALUES(69,'ad684b97-8b36-44bf-bb61-cad45ad1584d','2023-11-23 16:13:06.018509653+00:00','2607:2400:1106:200:54dc:f99a:9d34:a318','2023-11-23 16:18:06.018509798+00:00','dummyuser','ResetPassword','2023-11-23 16:13:13','2607:2400:1106:200:54dc:f99a:9d34:a318');
INSERT INTO identity_verification VALUES(70,'65f5f333-263e-4036-87da-edcb2935c5d6','2023-11-23 16:17:17.430066174+00:00','2607:2400:1106:200:54dc:f99a:9d34:a318','2023-11-23 16:22:17.43006656+00:00','dummyuser','RegisterTOTPDevice','2023-11-23 16:17:30','2607:2400:1106:200:54dc:f99a:9d34:a318');
INSERT INTO totp_configurations VALUES(125,'2023-11-23 16:17:30.378869666+00:00',NULL,'dummyuser','CDE:DCOM','SHA1',6,30,X'superlongstring');
INSERT INTO user_preferences VALUES(95,'dummyuser','totp');
INSERT INTO user_opaque_identifier VALUES(29,'openid','','dummyuser','daa0e907-b92f-4de1-bd78-80d7f6f6a320');
```

Im guessing `refresh_interval` is not "refreshing?" these tables.

Any insight is appreciated.

Thank you,
Dave

**Comment by james-d-elliott** [comment, marked as answer]
It has nothing to do with the database tables at all, it just refreshes the status and profile information from the backend. i.e. if their groups change or they are disabled or deleted it reflects that in their access.

**Comment by davama** [comment]
Thanks James!
